GitUp/Application/AppDelegate.m

```diff
@@ -275,6 +277,7 @@ - (id)_loadWindowFromBundleXibWithName:(NSString *)name expectedClass:(Class)cla
 }
 
 - (void)_loadWindowsFromBundle {
+  self.aboutPanel = [self _loadWindowFromBundleXibWithName:@"About" expectedClass:NSPanel.class];
```

Comment:
We should be loading these lazily on request. Can we please change the approach here to encapsulate xib loading in the class instead of having the AppDelegate have this logic?

Reply:
@lucasderraugh Do you want to load these windows on demand in `-(void)showWindow` methods?
By the way, should we also "inject" standardDefaults into them or shared instance of defaults would be fine?

Reply:
Either on showWindow or overriding the accessor to instantiate the window instance 1 time.

Reply:
Should get rid of this load with load on demand.